docs/maintainers/issue-management.md

```diff
@@ -1,8 +1,14 @@
 # Issue Management
 
+## Security Vulnerabilities
+
+If you believe that there is a security vulnerability in PowerShell Core,
+it **must** be reported to [secure@microsoft.com](https://technet.microsoft.com/en-us/security/ff852094.aspx) to allow for [Coordinated Vulnerability Disclosure](https://technet.microsoft.com/en-us/security/dn467923).
+**Only** file an issue, if secure@microsoft.com has confirmed filing an issue is appropriate.
```

Comment:
How can I find out that the approval is received? Should I delete messages that are not able to approve? Maybe the MSFT team would rather exclusively publish such Issues or even announcements?

Reply:
> How can I find out that the approval is received? 

For someone else, you cannot.  secure@microsoft.com would tell the person reporting the issue if it is appropriate to file an issue.   If this comes up, please alert one of the Microsoft maintainers.  We will report the vulnerability to secure@microsoft.com and take it from there.

> Should I delete messages that are not able to approve?

Don't delete a message yourself. 

> Maybe the MSFT team would rather exclusively publish such Issues or even announcements?

Yes, if it is found to actually be a vulnerability, most likely the filing of the issue will be done or at least coordinated by someone at Microsoft.  The only time I can see that secure@microsoft.com would say that they should file an issue, which is not coordinated with the release of the fix is when the issue is determined not to be a vulnerability.

I don't want to make these instructions too specific until we find a need.  The only need I see right now is if someone does report a vulnerability issue on GitHub, then, we can point to this policy on why we are not responding using GitHub.  Something to the effect of:

> This is not the appropriate place to report this issue, please see Security Vulnerabilities and report this issue to secure@microsoft.com.  I have reported this issue on your behalf in the meantime.

But, we will have to take it on a case by case basis and see what the most appropriate response is.

Reply:
Clear. Thanks!

/cc @markekraus Please read the new policy.